models/applicationSubmissionsModel.js

```diff
@@ -28,35 +29,22 @@ async function insert(submission) {
 
 async function update(id, decision) {
   return db('application_submissions')
-    .where({ id })
     .update({ decision })
+    .where({ id })
```

Comment:
Does order actually matter here? No big deal, no need to revert if not, I'm just curious.

Reply:
Here order does not matter. I changed it because it made more sense to me to follow the logic of selecting something to update where id = id (kind of like the UPDATE table SET value = x WHERE w = ? structure of a SQL query).